examples/basic/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
+
+pybind_extension(
+    name = "example",
+    srcs = ["binding.cpp"],
+    visibility = ["//visibility:public"],
+)
+py_library(
+    name = "add_py_lib",
+    data = [
+        "example.so",
+    ],
+    imports = ["."],
+    visibility = ["//visibility:public"],
+)
```

Comment:
```suggestion
py_library(
    name = "basic_lib",
    data = [":basic"],
    imports = ["."],
)
```

Reply:
Is it not best practice to include the visibility? I understand that since this is one build file, it is all in the same scope.

Reply:
I wouldn't bother setting it here, TBH, but if you do want to set it, then please set it on the `py_library` **_only_**.